File: Common_Character_Count/Common_character_count.java
import java.util.Arrays;
public class Common_character_count{
	public static void main (String[] args){
            String [][] tests = {
                    {"aabcc", "adcaa"},
                    {"a", "b"},
                    {"a", "aaa"}
            };
            for(String[] test: tests){
                System.out.format(" There are %d common characters between the words '%s' and '%s' %n%n", solution(test[0], test[1]), test[0], test[1]); 
            }
	}
	static int solution(String s1, String s2){
            int count = 0;
            for (char charr: s1.toCharArray())
                if(s2.contains(charr + "")){
                    s2.replaceFirst(charr + "", "");
                    count++; 
                }
           return count; 
        }
}

File: Common_Character_Count/Common_character_count.js
function solution(s1, s2) {
	let count = 0
	for(let character of s1){
		if(s2.includes(character)){
			s2 = s2.replace(character, "")
			count++
		}
	}

	return count;
}
const tests = [
	["aabcc", "adcaa"],
	["a", "b"],
	["a", "aaa"]
];

for (const test of tests) {
  console.log(` There are ${solution(test[0], test[1])} common characters between the words '${test[0]}' and '${test[1]} \n`);
}
